ocs_ci/helpers/ols_helpers.py

```
@@ -0,0 +1,672 @@
+    secret_file.close()
+    try:
+        secret_data = templating.load_yaml(constants.OLS_SECRET_YAML)
+        token = (
+            api_token
+            if api_token is not None
+            else ocsci_config.AUTH["ibm_watsonx_llm_for_ols"]["api_token"]
+        )
+        secret_data["stringData"]["apitoken"] = token
+        templating.dump_data_to_temp_yaml(secret_data, secret_path)
+        run_cmd(f"oc create -f {secret_path}")
+        return True
```

Ruff (0.15.15)

[warning] Consider moving this statement to an `else` block

(TRY300)